app/assets/javascripts/discourse/app/lib/user-menu/bookmark-item.js

```diff
@@ -24,7 +24,11 @@ export default class UserMenuBookmarkItem extends UserMenuBaseItem {
   }
 
   get label() {
-    return this.bookmark.user?.username;
+    if (this.siteSettings.prioritize_username_in_ux) {
+      return this.bookmark.user?.username;
+    }
+
+    return this.bookmark.user?.name || this.bookmark.user?.username;
```

Comment:
Same here, why do we default to the user's name instead of username when that was the previous behaviour.